src/main/java/com/iota/iri/network/Node.java

```diff
@@ -711,6 +711,17 @@ public Neighbor newNeighbor(final URI uri, boolean isConfigured) {
         return Optional.empty();
     }
 
+    private void parseNeighborsConfig() {
+        Arrays.stream(configuration.string(Configuration.DefaultConfSettings.NEIGHBORS).split(" ")).distinct()
+                .filter(s -> !s.isEmpty()).map(Node::uri).map(Optional::get)
+                .filter(u -> isUriValid(u))
```

Comment:
We should log invalid neighbours, could you create an issue?

Reply:
`isUriValid` logs the invalid entry: [Node.java:689](https://github.com/alon-e/iri/blob/948b8d0b5103903ec5445d474ac62ce3d4d2f6bb/src/main/java/com/iota/iri/network/Node.java#L689)